### PR TITLE
zee5 regex fix

### LINKED ISSUE #4612: [Zee5] HTTP Error 404 Issue.
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2022.08.08** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

Hello
I am trying to download from zee5 and I got HTTP Error 404 on this video link "https://www.zee5.com/kids/kids-shows/cinderella/0-6-2784/cindrella/0-1-manual_3osolr76hd4g"

Although some links are working and are downloadable for eg : "https://www.zee5.com/tv-shows/details/mithai/0-6-3121/mithai-july-17-2022/0-1-6z5177450"

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -v https://www.zee5.com/kids/kids-shows/cinderella/0-6-2784/cindrella/0-1-manual_3osolr76hd4g
[debug] Command-line config: ['-v', 'https://www.zee5.com/kids/kids-shows/cinderella/0-6-2784/cindrella/0-1-manual_3osolr76hd4g']
[debug] User config "C:\Users\user\AppData\Roaming\yt-dlp\config.txt": ['--all-subs', '--embed-subs', '--embed-thumbnail', '--add-metadata', '--write-annotations', '--download-archive=All Youtube History.txt', '--write-annotations', '--write-description', '--write-info-json', '--write-thumbnail', '-f', 'bv+ba/best', '--merge-output-format', 'mp4', '-ciw', '--retries', 'infinite', '-o', '%(title)s-%(id)s-%(format_id)s.%(ext)s']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.08.08 [3157158] (win32_exe)
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19043-SP0
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg 4.4-full_build-www.gyan.dev (setts), ffprobe 4.4-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loading archive file 'All Youtube History.txt'
[debug] Using fake IP 117.227.126.177 (IN) as X-Forwarded-For
[debug] [Zee5] Extracting URL: https://www.zee5.com/kids/kids-shows/cinderella/0-6-2784/cindrella/0-1-manual_3osolr76hd4g
[Zee5] 0-1-manual_3osolr76hd4g: Downloading access token
[Zee5] 0-1-manual_3osolr76hd4g: Downloading JSON metadata
[Zee5] 0-1-manual_3osolr76hd4g: Downloading m3u8 information
WARNING: [Zee5] Failed to download m3u8 information: HTTP Error 404: Not Found
ERROR: [Zee5] 0-1-manual_3osolr76hd4g: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
Traceback (most recent call last):
  File "yt_dlp\YoutubeDL.py", line 1441, in wrapper
  File "yt_dlp\YoutubeDL.py", line 1538, in __extract_info
  File "yt_dlp\YoutubeDL.py", line 1596, in process_ie_result
  File "yt_dlp\YoutubeDL.py", line 2514, in process_video_result
  File "yt_dlp\YoutubeDL.py", line 1009, in raise_no_formats
yt_dlp.utils.ExtractorError: [Zee5] 0-1-manual_3osolr76hd4g: No video formats found!; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
```

### PR DESCRIPTION
**Pull request by m4tu4g**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

</details>

Issue with regex for matching kids-movies

Fixes #4612 

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### CODE REVIEW

File: yt_dlp/extractor/zee5.py [hunk 1]
```diff
@@ -176,7 +179,7 @@ class Zee5SeriesIE(InfoExtractor):
                      (?:
                         zee5:series:|
                         https?://(?:www\.)?zee5\.com/(?:[^#?]+/)?
-                        (?:tv-shows|web-series|kids|zee5originals)(?:/[^#/?]+){2}/
```
Comment (pukkandan) [inline yt_dlp/extractor/zee5.py:179]: why not just `[^#/?]+`?

Comment (m4tu4g) [reply yt_dlp/extractor/zee5.py:179]: yup, old one seems fine.

Comment (m4tu4g) [reply yt_dlp/extractor/zee5.py]: now there is a conflict of "kids-movies" matching zee5seriesIE regex

File: yt_dlp/extractor/zee5.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Zee5IE(InfoExtractor):
                         https?://(?:www\.)?zee5\.com/(?:[^#?]+/)?
                         (?:
                             (?:tv-shows|kids|web-series|zee5originals)(?:/[^#/?]+){3}
-                            |movies/[^#/?]+
+                            |(?:movies|kids|videos)/(?:details|kids-movies)+
```
Comment (pukkandan) [inline yt_dlp/extractor/zee5.py:26]:
```suggestion
                            |(?:movies|kids|videos)/[^#/?]+
```

**Pull request merged**